Library/docs/source/learn/overview/tenants.md

```diff
@@ -0,0 +1,152 @@
+Tenants are necessary for securely organizing and managing resources. They help to isolate data and applications for users.
+
+## What is an MCS Tenant?
+
+An MCS (Magnopus Cloud Services) tenant is a dedicated instance within the services backend specifically designed to segregate users and their data from others. Each MCS tenant operates independently, ensuring that data and interactions remain isolated.
+
+A developer building a CSP-powered application requires their own tenant to ensures that users of their application cannot interact with users of a different CSP-powered application.
+
+## Multi-Tenancy in MCS
+
+Multi-tenancy is an architecture where a single cloud hosted environment  and its supporting infrastructure are shared among two or more tenants at the same time. While each tenant’s data is isolated from the other tenants, each tenant shares a single database and SaaS server with the others.
+
+With MCS, this means that one environment can host multiple tenants, each with its own segregated data. 
+
+**Benefits for Clients:**
+
+* **Cost Savings:** Services are affordable as clients share the underlying infrastructure costs.
+
+* **Customizability:** Each tenant can customize its environment to meet specific needs.
+
+* **Security:** Isolation between tenants ensures that one tenant's data and activities do not affect another's.
+
+## Requesting an MCS Tenant
+
+### Prerequisites for Requesting a Tenant
+
+Before requesting an MCS (Magnopus Cloud Services) tenant, ensure you have:
+
+* **Company Name and Email:** A company name and email address is required. Personal email addresses are not accepted.
+
+### Step-by-Step Process to Request an MCS Tenant
+
+As a developer looking to use CSP and Magnopus Cloud Services, you must fill out an online form to request an MCS tenant. Follow these steps to complete your request:
+
+1. Go to [Magnopus CSP for Developers](https://www.magnopus.com/csp/for-developers#tenant-id).  
+2. Fill out the **Tenant Inquiry Form**  
+3. Review all the information you entered to ensure accuracy and click the **Submit Inquiry** button to send your request. You will receive a confirmation email acknowledging receipt of your request.  
+4. Tenant IDs requests are reviewed every two weeks, in line with Magnopus’ release schedule.  
+   If you have not heard back from the Magnopus team in two weeks confirming the creation of your tenant, please email [tenantkey@magnopus.com](mailto:tenantkey@magnopus.com).
+
+## Initializing CSP with a Tenant
+
+### How to Direct CSP to a Tenant
+
+Tenants are uniquely identified by a string that Magnopus will share with you once your tenancy request has been approved. 
+
+To direct CSP to a tenant, you need to specify the tenant string during the initialization of your application. This involves providing the tenant as an argument to the csp::CSPFoundation::Initialise function. This is expected to be the very first CSP invocation a client application makes in it’s lifecycle. 
```

Comment:
I don't want to delay this PR with another bunch of nitpicks that likely only professional editors and grammar enthusiasts like me see, however please do fix this... "its lifecycle". (not a verb! :eyes:)

BTW, if translating this in other languages is ever on the table, I am more than happy to help with the French version.

Reply:
I might take you up on that! Fixed the typo on latest 👍